src/tools/segmentation/utils/PointInside/pointInPolygon.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Checks whether a point is inside a polygon
```

Comment:
We basically already have this implemented under `util/freehand/pointInFreehand.js`.

We should only have one implementation, however feel free to benchmark which is faster and replace the existing method if needbe.

Reply:
Hey @JamesAPetts and @swederik  we left both implementations on the code, just commented, the new one and the old one from pointInFreehand.j to bench.

The old one seems not to be working properly. 

Please see the attached image:

We can discuss this tomorrow together to see if we made something wrong.

![freeHanDPointInPolygon](https://user-images.githubusercontent.com/3472146/60466100-22073280-9c29-11e9-8159-e238d11a7b8a.gif)

Reply:
The exact implementation of `pointInFreehand` might not be exactly what it is needed, I more meant that they are very similar in premise and some refactoring of common code might be possible.

This point is minimal and I wouldn't worry about it too much. If your current implementation is working well, I would focus on just getting that tied up.